bin/create-bioentities-suggesters.sh

```diff
@@ -14,10 +14,7 @@ COLLECTION=${SOLR_COLLECTION:-"bioentities-v${SCHEMA_VERSION}"}
 
 printf "\n\nDelete search component for suggesters...\n"
 curl $SOLR_AUTH -X POST -H 'Content-type:application/json' --data-binary '{
-  "delete-searchcomponent": {
-    "name": "suggest",
-    "class": "solr.SuggestComponent"
-  }
+  "delete-searchcomponent": "suggest"
```

Comment:
Remove this extra white space please before merging

Reply:
ex:

'{
  "delete-searchcomponent": "suggest"
'''''''''''' no extra white space here 
'''''''''''''' ---> no extra space here 
}'

Reply:
Ignore the above comment if there is no extra white space between the parenthesis of the JSON body.